src/components/ThumbsFeedback.tsx

```diff
@@ -1,8 +1,11 @@
 import { useSearchState } from '@yext/search-headless-react';
 import { useCallback, useState } from 'react';
 import { ThumbIcon } from '../icons/ThumbIcon';
-import useLayoutEffect from 'use-isomorphic-layout-effect';
 import { useComposedCssClasses } from '../hooks/useComposedCssClasses';
+import useIsomorphicLayoutEffect from "use-isomorphic-layout-effect";
```

Comment:
I recommend opening an issue with this library. It'd be very easy for them to fix their ESM outputs and would avoid you having to do this bologna.

https://publint.dev/use-isomorphic-layout-effect@1.1.2

Reply:
We could also just fork the library and fix it in our fork, and submit a PR

Reply:
Would that be the same for the rest of the bologna?

Reply:
It would mean you should be able to use named imports properly instead of a package-level default import and getting the properties off of it. And also not have to do these weird checks. At least I'd hope not. It might depend on their actual implementation of what they're exporting.